homeassistant/components/homekit/util.py

```diff
@@ -36,7 +37,8 @@
 MEDIA_PLAYER_SCHEMA = vol.Schema({
     vol.Required(CONF_FEATURE): vol.All(
         cv.string, vol.In((FEATURE_ON_OFF, FEATURE_PLAY_PAUSE,
-                           FEATURE_PLAY_STOP, FEATURE_TOGGLE_MUTE))),
+                           FEATURE_PLAY_STOP, FEATURE_TOGGLE_MUTE,
+                           FEATURE_SELECT_SOURCE, FEATURE_VOLUME_STEP))),
```

Comment:
This change can probably be reverted. Same below inside `validate_media_player_features`.